src/foam/u2/borders/TopBorderCard.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'TopBorderCard',
+  extends: 'foam.u2.View',
+
+  css: `
+    ^ {
+      width: 100%;
+      border: 1px solid/*%GREY5%*/ #f5f7fa;
+    }
+    ^bar {
+      width: 100%;
+      height: 8px;
+      border-top-left-radius: 4px;
+      border-top-right-radius: 4px;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'Color',
+      name: 'color'
+    }
+  ],
+
+  methods: [
+    function init() {
+      this.start().addClass(this.myClass('bar')).end()
+      .start('div', null, this.content$)
+        .addClass()
+        .style({ 'background' : this.color$,
+                 'border' : '1px solid' + this.color$ })
```

Comment:
Shouldn't this border prop be on the div with class `bar`

Reply:
yes, you are right. my bad. I will fix this. thank you.